setuptools/__init__.py

```diff
@@ -165,8 +169,9 @@ class Command(_Command):
     """
 
     command_consumes_arguments = False
+    distribution: Distribution  # override distutils.dist.Distribution with setuptools.dist.Distribution
 
-    def __init__(self, dist, **kw):
+    def __init__(self, dist: Distribution, **kw):
```

Comment:
I think one of these two lines is causing the doc workflow to fail, but I don't understand why.

Reply:
Is this something that we can workaround by adding an exception to the `nitpick` configuration in `docs/conf.py`?